cli/src/cmd/forge/create.rs

```diff
@@ -147,21 +151,14 @@ impl Cmd for CreateArgs {
         // Deploy with signer
         let rt = tokio::runtime::Runtime::new().expect("could not start tokio rt");
         let chain_id = rt.block_on(provider.get_chainid())?;
-        if let Some(signer) = rt.block_on(self.eth.signer_with(chain_id, provider))? {
-            match signer {
-                WalletType::Ledger(signer) => {
-                    rt.block_on(self.deploy(abi, bin, params, signer))?;
-                }
-                WalletType::Local(signer) => {
-                    rt.block_on(self.deploy(abi, bin, params, signer))?;
-                }
-                WalletType::Trezor(signer) => {
-                    rt.block_on(self.deploy(abi, bin, params, signer))?;
-                }
-            }
-        } else {
-            eyre::bail!("could not find artifact")
-        }
+        match rt.block_on(self.eth.signer_with(chain_id, provider))? {
```

Comment:
I would rather make the entire function async then dealing with the rt at the cmd level, I believe we do that for verify already